collections/_article/dev-snapshot-godot-4-4-dev-5.md

```diff
@@ -0,0 +1,135 @@
+So buckle on in for a breakdown of what you can expect to see in dev5.
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also try the [**Web editor**](https://editor.godotengine.org/releases/4.4.dev5/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/), or the **Android editor** for this release (join the [Android editor testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds).
+
+-----
+
+*The cover illustration is from* [**The Rise of the Golden Idol**](https://store.steampowered.com/app/2716400/The_Rise_of_the_Golden_Idol/), *a mystery game where you unravel the truth behind 20 cases of crime in the 1970s! It is developed by [Color Gray Games](https://www.thegoldenidol.com/). You can purchase the game [on Steam](https://store.steampowered.com/app/2716400/The_Rise_of_the_Golden_Idol/) and follow the developer on [Twitter](https://twitter.com/colorgray7/).*
+
+## Highlights
+
+In case you missed them, see the [4.4 dev 1](/article/dev-snapshot-godot-4-4-dev-1/), [dev2](/article/dev-snapshot-godot-4-4-dev-2/),
+[dev 3](/article/dev-snapshot-godot-4-4-dev-3/), and [dev4](/article/dev-snapshot-godot-4-4-dev-4/) release notes for an overview of
+some key features which were already in that snapshot, and are therefore still available for testing in dev 5.
+
+Here are a few highlights that you might find particularly exciting!
+
+### Universalize UID support
+
+Previously, the <abbr title="Universal ID">UID</abbr> format was limited to only Resource files. This proved to be a pain point for users wishing to reference their scripts and other resources in a manner that's path-agnostic. This, along with a need refactor their scripts anytime these kinds of files were moved, put a significant duty of care on the end-user that shouldn't have been necessary.
+
+Starting with dev5, this will no longer be your burden to bear! Thanks to a long-term effort from [reduz](https://github.com/reduz), UIDs will now be applied universally in a way the engine can automatically track and account for. This is achived via `.uid` files for the resource types that previously didn't support them, functioning similarly to other metadata files that "track" a main file.
```

Comment:
```suggestion
Starting with dev 5, this will no longer be your burden to bear! Thanks to a long-term effort from [reduz](https://github.com/reduz), UIDs will now be applied universally in a way the engine can automatically track and account for. This is achieved via `.uid` files for the resource types that previously didn't support them, functioning similarly to other metadata files that "track" a main file.
```